app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<section id="all_tasks">
+<h2>Do these things:</h2>
+<ol>
+  <% @tasks.each do |task| %>
+    <section id="list-tasks">
+    <% if task.completed == false %>
+      <li><strong><%= link_to "Task: #{task.name}", task_path(task.id) %></strong></li>
+    <% elsif task.completed == true %>
+      <li id="crossed-off"><%= link_to "Task: #{task.name}", task_path(task.id) %></li>
+    <% end %>
```

Comment:
This logic looks great! However, there's a pretty good opportunity for refactoring here. Instead of `task.completed == false`, it might be nice to just evaluate on`!task.completed` or `task.completed`, or maybe even use an `if ... else` instead of `if ... elsif`